packages/strapi-tiptap-editor/admin/src/components/Editor/index.tsx

```diff
@@ -68,25 +69,39 @@ const Editor = ({ editor, settings, productPrice }: EditorProps) => {
   const availableThemes: Theme[] = ['light', 'dark'];
   const isTheme = (arg: any): arg is Theme => availableThemes.some((x) => x === arg);
   const localStorageTheme = getLocalStorage('STRAPI_THEME', isTheme);
-
   return (
     <Wrapper>
-      <Box hasRadius overflow="hidden" borderWidth="1px" borderStyle="solid" borderColor="neutral200">
-        <Toolbar
-          editor={editor}
-          toggleMediaLib={handleToggleMediaLib}
-          settings={settings}
-          productPrice={productPrice}
-        />
-        <BubbleMenuComponent editor={editor} />
+      <StickyContainer>
+        <Sticky topOffset={-50}>
+          {({ style }) => {
+            console.log(style);
+
+            return (
+              <div
+                style={{
+                  ...style,
+                  zIndex: 10,
+                }}
+              >
+                <Toolbar
+                  editor={editor}
+                  toggleMediaLib={handleToggleMediaLib}
+                  settings={settings}
+                  productPrice={productPrice}
+                />
+                <BubbleMenuComponent editor={editor} />
```

Comment:
Hoort de `BubbleMenuComponent` component ook in `StickyContainer`?

Reply:
Nee, Ik heb het uitgehaald van de `StickyContainer `.